x-pack/plugins/security_solution/public/timelines/components/side_panel/host_details/index.tsx

```diff
@@ -80,7 +82,9 @@ export const HostDetailsPanel: React.FC<HostDetailsProps> = React.memo(
         <StyledEuiFlyoutBody>
           <EuiSpacer size="m" />
           <ExpandableHostDetailsPageLink hostName={hostName} />
-          <EuiSpacer size="m" />
+          <EuiHorizontalRule />
```

Comment:
Both `EuiHorizontalRule ` will render when asset criticality flag is disabled, or the user doesn't have the privileges.

Reply:
updated